cs/test/comm/Epoxy/EpoxyListenerTests.cs

```diff
@@ -120,6 +122,25 @@ public class EpoxyListenerTests : EpoxyTestBase
             Assert.AreEqual(connection.LocalEndPoint, disconnectedConnection.RemoteEndPoint);
         }
 
+        [Test]
+        public async Task OneConnectionStalledDuringHandshake_CanAcceptAnother()
+        {
+            EpoxyTransport transport = MakeTransport();
+            listener = transport.MakeListener(localhostEndpoint);
+            await listener.StartAsync();
+
+            var noHandshakeConnection = new TcpClient();
+            // This will just establish a TCP connection. It won't perform the Epoxy handshake, so
+            // the listener will just be sitting there waiting for the client to send some data.
+            await noHandshakeConnection.ConnectAsync(localhostEndpoint.Address, localhostEndpoint.Port);
+
+            var connectTask = transport.ConnectToAsync(localhostAddress);
+            bool didConnect = connectTask.Wait(TimeSpan.FromSeconds(30));
```

Comment:
This is a huge amount of time. I would think (hope?) this test would be reliable with a fraction of a second.